src/lib/fetchers.tsx

```diff
@@ -112,6 +112,7 @@ export interface Seat {
 export interface KualiSubject {
   subject: string;
   title: string;
+  inSession?: boolean;
```

Comment:
This file isn't stable (it's generated) so I suggest making a type that extends this one instead.